devices.js

```diff
@@ -15930,16 +15930,18 @@ const devices = [
         toZigbee: [tz.thermostat_local_temperature, tz.thermostat_occupied_heating_setpoint, tz.thermostat_control_sequence_of_operation,
             tz.thermostat_system_mode, tz.thermostat_keypad_lockout],
         exposes: [exposes.climate().withSetpoint('occupied_heating_setpoint', 7, 30, 1).withLocalTemperature()
-            .withSystemMode(['off', 'auto', 'heat']).withRunningState(['idle', 'heat'], ea.STATE)],
-        meta: {configureKey: 1},
+            .withSystemMode(['heat', 'sleep'])],
```

Comment:
Can sleep be renamed to `off`? HA doesn't support this.

Reply:
I don't think we should: https://github.com/Koenkk/zigbee-herdsman-converters/blob/67fd494874dbc5e14998a2252b5942a3bf2b63c4/lib/constants.js#L33

It's set to 9, which is sleep according to the ZCL too. This seems like something to do in the hass plugin instead.

Reply:
I see, thanks

Reply:
I'm also not sure if this has other effects yet. What I observed:
TRV as set to heat and actually heating the room from 20 -> 24C

Set it to sleep: 1. closed valve and stopped heating, 2. reporting seems to happen less frequent.

I guess it might be waiting for it to hit max reporting timer... or due to it not heating localTemp changed less often... that could also be the case. Although the room still got warmer due to the radiator still being hot.

Reply:
@Koenkk would something like

https://github.com/Koenkk/zigbee2mqtt/blob/52b80221730b7673e98a54385ea4c6ec332db850/lib/extension/homeassistant.js#L166

```
if (mode.values.includes('sleep')) {
 // remove 'sleep' from mode.values ?
}
```

Not sure there is a javascript/node version of pythons `mylist.remove('sleep')`, basically if hass.io doesn't do sleep, we can remove it there right? That should still allow things like the frontend or other expose consumers to set 'sleep'.

And hass.io in this case will only have 'heat'.

Reply:
Yes, that's fine for me. Could you make a PR?

Reply:
I could, but I don't use hass.io so can't really test it. Will be tomorrow though.